src/features/myAccounts/hooks/useTrackedSafesCount.ts

```diff
@@ -46,6 +42,15 @@ const useTrackSafesCount = (
     [safes, watchlistMultiChainSafes],
   )
 
+  // TODO: This is computed here and inside PinnedSafes now. Find a way to optimize it
```

Comment:
A small improvement could be to extract the calculations of allSafes and pinnedSafes into helper functions since they are both done twice, but nbd. IMO it's ok as is